lms/static/sass/_build-lms-v2.scss

```diff
@@ -19,7 +19,10 @@
 @import 'shared-v2/modal';
 @import 'shared-v2/help-tab';
 
+// Elements
 @import 'notifications';
+@import 'elements-v2/pagination';
 
-// course outline
-@import 'shared-v2/course-outline';
+// Features
+@import 'features/bookmarks';
```

Comment:
The Django app is called course_bookmarks, but I left files such as this the way they were (I just moved the file into a subdirectory).